braze-templates/4-sms-capture-modal/README.md

```diff
@@ -1,3 +1,5 @@
+> ⚠️ Note: Subscription Groups were introduced in Android SDK v15.0.0, Web SDK v3.4.0, and iOS SDK v4.3.3. Please ensure your SDKs are up to date, or apply filters to your message's targeting.
```

Comment:
@nickrobin how's this?

Reply:
OK - that works for me ! Thanks for pulling the exact versions!